sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedReceiptItem.cs

```diff
@@ -13,9 +15,20 @@ internal ExtractedReceiptItem(string name, int? quantity, float? price, float? t
             TotalPrice = totalPrice;
         }
 
+        /// <summary>
+        /// </summary>
         public string Name { get; }
+
```

Comment:
missing ItemPrice

Reply:
Is that different from [Price ](https://github.com/Azure/azure-sdk-for-net/pull/10562/files/f099fbf5a88b8fcde62cde78dd59002c8a744470#diff-82c767c4c0236e82a7a1cee129927b66R28)below?

Reply:
Sorry, missed that one!